commit a6cf450e02c0a6a7178e2f8fb9d56bfd829e012b
Author: Greg Kroah-Hartman <gregkh@suse.de>
Date:   Sun Oct 10 05:52:59 2010 -0700

    Staging: brcm80211: remove __cplusplus markers
    
    It's not needed within the kernel, so remove them.
    
    Cc: Brett Rudley <brudley@broadcom.com>
    Cc: Henry Ptasinski <henryp@broadcom.com>
    Cc: Nohee Ko <noheek@broadcom.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

--- a/drivers/staging/brcm80211/include/bcmutils.h
+++ b/drivers/staging/brcm80211/include/bcmutils.h
@@ -17,10 +17,6 @@
 #ifndef	_bcmutils_h_
 #define	_bcmutils_h_
 
-#ifdef __cplusplus
-extern "C" {
-#endif
-
 /* Buffer structure for collecting string-formatted data
 * using bcm_bprintf() API.
 * Use bcm_binit() to initialize before use
@@ -513,8 +509,4 @@ extern "C" {
 				uint len);
 	extern uint bcm_bitcount(u8 *bitmap, uint bytelength);
 
-#ifdef __cplusplus
-}
-#endif
-
 #endif				/* _bcmutils_h_ */
